content/en/service_management/app_builder/_index.md

```diff
@@ -59,6 +59,12 @@ To use the app with your data, click **+ Connect Data**, then either create a ne
 
 You can change the selected connection by clicking **Change Connection** in the app.
 
+## App Builder Overview dashboard
```

Comment:
the dashboard is called "App Builder Overview" so i think this capitalization is ok, but calling out cause linting disagrees

Reply:
Yes, I agree, as this is the title of the dashboard.